inspirehep/records/cli.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of INSPIRE.
+# Copyright (C) 2014-2017 CERN.
+#
+# INSPIRE is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# INSPIRE is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with INSPIRE. If not, see <http://www.gnu.org/licenses/>.
+#
+# In applying this license, CERN does not waive the privileges and immunities
+# granted to it by virtue of its status as an Intergovernmental Organization
+# or submit itself to any jurisdiction.
+
+import click
+from flask.cli import with_appcontext
+
+from inspirehep.records.fixtures import (
+    init_default_storage_path,
+    init_records_files_storage_path,
+)
+
+
+@click.group()
+def fixtures():
+    """Command related to records in inspire"""
+
+
+@fixtures.command()
+@with_appcontext
+def init_records_store_paths():
+    """Init default file and records store location."""
+    init_default_storage_path()
+    init_records_files_storage_path()
```

Comment:
we can move it to ``cli/fixutres.py`` so we can register each ``group`` independently